contribute/architecture/services.md

```diff
@@ -0,0 +1,183 @@
+# Services
+
+A Grafana _service_ encapsulates and exposes application logic to the rest of the application, through a set of related operations. 
+
+Services are self-contained, and only talk to services using a [service bus](#service-bus) or repositories. Before a service can start communicating with the rest of Grafana, it needs to be registered in the _service registry_. The service registry keeps track of all available services during runtime.
+
+## Your first service
+
+Even though the services in Grafana do different things, they share a number of patterns. To better understand how a service works, let's build one from scratch!
+
+### Create a service
+
+To start building a service:
+
+- Create a new Go package `mysvc` in the [pkg/services](/pkg/services) directory.
+- Create a `service.go` file inside your new directory.
+
+All services need to implement the [Service](https://godoc.org/github.com/grafana/grafana/pkg/registry#Service) interface:
+
+```go
+type MyService struct {
+}
+
+func (s *MyService) Init() error {
+    return nil
+}
+```
+
+The `Init` method is used to initialize and configure the service to make it ready to use.
```

Comment:
```suggestion
The `Init` method is used to initialize and configure the service to make it ready to use. Services that return an error will halt Grafana's startup process and cause the error to be logged as it exits.
```

Reply:
```suggestion
The `Init` method is used to initialize and configure the service to make it ready to use. Services that return an error halts Grafana's startup process and cause the error to be logged as it exits.
```

Minor change to keep consistent present tense.